stats/series_test.go

```diff
@@ -0,0 +1,56 @@
+package stats
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIntSeries(t *testing.T) {
+	type results struct {
+		len uint
+		min, max, total, avg int
+		str string
+	}
+	tests := []struct{
+		name string
+		samples []int
+		format string
+		want results
+	}{
+		{
+			name: "No samples",
+			samples: []int{},
+			format: "%v",
+			want: results{str: "min: 0 max: 0 avg: 0"},
+		},
+		{
+			name: "3 samples",
+			samples: []int{2, 8, 8},
+			format: "Stats: %3d",
+			want: results{
+				len: 3,
+				min: 2,
+				max: 8,
+				total: 18,
+				avg: 6,
+				str: "Stats: min:   2 max:   8 avg:   6",
```

Comment:
Is there a reason we want 3 spaces here?

Reply:
to test that the format string works.

Reply:
What I mean is, why do we want the format to have 3 spaces in the first place

Reply:
see how this helps to align output in the place where we use this as a part of the uploader test.